app/ide-desktop/lib/content/src/index.ts

```diff
@@ -772,6 +772,7 @@ class Config {
     public node_labels: boolean = true
     public crash_report_host: string = defaultLogServerHost
     public data_gathering: boolean = true
+    public mixpanel_token: string = undefined
```

Comment:
Actually, since the token is not secret, we can put it here as a default value. Then we can remove the user-facing `mixpanel-token` CLI option, while still being able to override it in the cloud